src/updater-builder.ts

```diff
@@ -55,12 +54,10 @@ export class UpdaterBuilder {
         `UPDATER_ONE_CONTAINER=1`,
         `ENABLE_CONNECTIVITY_CHECK=1`
       ],
-      User: 'root',
       Cmd: ['sh', '-c', cmd],
       HostConfig: {
         Memory: UPDATER_MAX_MEMORY,
-        NetworkMode: this.proxy.networkName,
-        Binds: [`${this.outputHostPath}:${JOB_OUTPUT_PATH}:rw`]
```

Comment:
Was this bind only being used for running update-ca-certificates?

Reply:
This was leftover from https://github.com/github/dependabot-action/pull/319. Since I removed `User: "root"` this started failing because the bind is owned by root. It's not used for anything anymore due to #319 so I just deleted it.

Reply:
>No need to read the output file and turn it into an input file for the next step.